spec/index.bs

```diff
@@ -36,6 +36,10 @@ spec: webidl; urlPrefix: https://heycam.github.io/webidl/#
 spec: web-apis; urlPrefix: https://html.spec.whatwg.org/multipage/webappapis.html#
     type: dfn
         text: cross-origin isolated; url: cross-origin-isolated
+spec: canvas; urlPrefix:https://html.spec.whatwg.org/multipage/canvas.html#
```

Comment:
HTML spec is included by default. Played with this a bit - by using `{{HTMLCanvasElement/getContext()}}` no custom import is needed. I pushed a commit to do this.

Reply:
awesome, TIL, thanks!